flask_restful/__init__.py

```diff
@@ -400,7 +400,11 @@ def _register_view(self, app, resource, *urls, **kwargs):
         self.endpoints.add(endpoint)
 
         if endpoint in app.view_functions.keys():
-            previous_view_class = app.view_functions[endpoint].__dict__['view_class']
+            try:
```

Comment:
I often run into conflicts during dev where two things have the same route. Without the try except it cause an error but you can't tell why there was a problem